### DISCUSSION TITLE
ToolDiy/software/gdbdashboard/

### DISCUSSION THREAD

**Discussion by giscus** [cargo-youth/ToolDiy#24] [Announcements]
# ToolDiy/software/gdbdashboard/

https://cargo-youth.github.io/ToolDiy/software/gdbdashboard/

**Comment by storage-db** [comment]
这个插件使用问题可以后续一起讨论，我在内核debug的学习中也学习了解到不少很厉害的指令

**Comment by pluveto** [comment]
Cool!

**Comment by jackyliu16** [comment]
找到一篇还不错的[gdb文档](https://wizardforcel.gitbooks.io/100-gdb-tips/content/part1.html)大家可以看下

**Reply by storage-db** [reply]
感谢
